docs/reference/es_compatible_api.md

```diff
@@ -131,10 +132,47 @@ If a parameter appears both as a query string parameter and in the JSON payload,
 | `from`     | `Integer`   |  The rank of the first hit to return. This is useful for pagination.  |  0  |
 | `query` | `Json object` | Describe the search query. See [Query DSL](#query-dsl) | (Optional) |
 | `size` | `Integer` | Number of hits to return. |  10 |
-| `sort` | `JsonObject[]` | Describes how documents should be ranked. | `[]` |
+| `sort` | `JsonObject[]` | Describes how documents should be ranked. See [Sort order](#sort-order) | `[]` |
 | `aggs` | `Json object` | Aggregation definition. See [Aggregations](aggregation.md). | `{}` | `
 
 
+#### Sort order
+
+You can define up to two criteria on which to apply sort.
+If two criteria are supplied, the lexicographical order is applied.
```

Comment:
This phrase is open for interpretation if you have two text fields and can cause some confusion. I would just omit it and go straight to describing how the second field is use as a tie breaker if the first field have the same value in multiple documents.